hummingbird/ml/operator_converters/linear_classifier.py

```diff
@@ -0,0 +1,98 @@
+# -------------------------------------------------------------------------
```

Comment:
Should we call the file `linear` rather then `linear_classifier`? We do regression, not just classification.

Reply:
done